contrib/unbound.service.in

```diff
@@ -63,8 +63,8 @@ ProtectHome=true
 ProtectControlGroups=true
 ProtectKernelModules=true
 ProtectSystem=strict
-RuntimeDirectory=unbound
-ConfigurationDirectory=unbound
+RuntimeDirectory=@UNBOUND_RUN_DIR@
```

Comment:
In unbound [UNBOUND_RUN_DIR](https://github.com/NLnetLabs/unbound/blob/a45a7efe694d7524d2d37070b5266453004bd4a1/configure.ac#L181) points to where config is stored (also default workoing directory) which is usually `/etc/unbound`. systemd RuntimeDirectory points to path under `/run` so those aren't related beside similar name.

Reply:
Unless I'm mistaken, all of these directories must be relative.

Excerpt from https://www.freedesktop.org/software/systemd/man/systemd.exec.html#RuntimeDirectory=:
> RuntimeDirectory=, StateDirectory=, CacheDirectory=, LogsDirectory=, ConfigurationDirectory=
> 
> These options take a whitespace-separated list of directory names. **The specified directory names must be relative, and may not include "..".**

Reply:
Right, which means it's not even possible to set it this way.